docs/self-managed/identity/getting-started/install-identity.md

```diff
@@ -21,6 +23,9 @@ Navigate to [localhost:8080](http://localhost:8080) to see the UI exposed by Ide
 
 The configuration in this guide creates an example user during installation; use this account to log in:
 
+<!-- Is this true for any setup? -->
+<!-- For setting up production systems, we should nudge users to remove this configuration prior to launching. -->
```

Comment:
Do we need to add removing the demo user from https://docs.camunda.io/docs/self-managed/identity/user-guide/configuration/making-identity-production-ready/?

Reply:
The demo user is included [in the helm charts](https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml#L1373) (it seems that this is the default case). Therefore it would make sense to also have it documented in the helm installation part of the docs. What do you think?

Reply:
Interesting - we should surely mention that the demo/demo user is created by default in the Helm chart. 

Do we need this page, then? Could we consolidate it with the "[What is Identity](https://docs.camunda.io/docs/self-managed/identity/what-is-identity/)" page and walk through what screens are available in the Identity UI?